rcl/src/rcl/wait.c

```diff
@@ -64,6 +64,7 @@ static void
 __wait_set_clean_up(rcl_wait_set_t * wait_set, rcl_allocator_t allocator)
 {
   rcl_ret_t ret;
+  (void)ret;  // NO LINT
```

Comment:
No particular reason, but I don't see an advantage to changing that.

As each of the asserts are commented those are defensive checks, so I'd say they don't need to be checked at runtime. I'm sort of agnostic one way or the other.